node/core/av-store/src/lib.rs

```diff
@@ -582,7 +597,13 @@ async fn process_block_activated<Context>(
 where
 	Context: SubsystemContext<Message=AvailabilityStoreMessage>
 {
-	let events = request_candidate_events(ctx, hash).await?;
+	let events = match request_candidate_events(ctx, hash).await {
+		Ok(events) => events,
+		Err(err) => {
+			log::debug!(target: LOG_TARGET, "requesting candidate events failed due to {}", err);
+			return Ok(());
+		}
```

Comment:
This would be a common thing too I assume? Could be a generic `fn log_and_drop<E: ..., R: Try<..>>(e: E) -> R`